bedrock/firefox/templates/firefox/firstrun/firstrun-e.html

```diff
@@ -0,0 +1,104 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% add_lang_files "firefox/whatsnew_38" %}
+
+{% extends "firefox/base.html" %}
+
+{% block extra_meta %}<meta name="robots" content="noindex">{% endblock %}
+
+{#- This will appear as <meta property="og:title"> which can be used for social share -#}
+{% block page_og_title %}
+  {{_('Choose the independent browser')}}
+{% endblock %}
+
+{#- This will appear as <meta property="og:description"> which can be used for social share -#}
+{% block page_og_desc %}
+  {{_('Firefox is non-profit, non-corporate, non-compromised. Choosing Firefox isn’t just choosing a browser. It’s a vote for personal freedom.')}}
+{% endblock %}
+
+{#- Override <meta property="og:url"> for social share -#}
+{% block page_og_url %}{{ url('firefox') }}{% endblock %}
+
+{% block body_class %}mozID variant-e{% endblock %}
+
+{% block site_css %}
+  {% stylesheet 'firefox_variants_firstrun' %}
+{% endblock %}
+
+{% block optimizely %}
+  {% if waffle.switch('firefox-firstrun-optimizely') %}
+    {% include 'includes/optimizely.html' %}
+  {% endif %}
+{% endblock %}
+
+{% block site_header %}
+  <header id="masthead">
+    {% block tabzilla_tab %}
+    <div id="tabzilla">
+      <a href="{{ url('mozorg.home') }}">Mozilla</a>
+    </div>
+    {% endblock %}
+    {% block site_header_nav %}{% endblock %}
+    {% block site_header_logo %}{% endblock %}
+    {% block breadcrumbs %}{% endblock %}
+  </header>
+{% endblock %}
+
+{% block content %}
+<section id="intro" data-fxa-iframe-src="{{ settings.FXA_IFRAME_SRC }}">
+  <div class="container">
+    <div id="logo">
+      {{ high_res_img('firefox/australis/fx38.0.5/firefox-logo.png', {'alt': 'Firefox', 'width': '142', 'height': '147'}) }}
+    </div>
+
+    {#  form left #}
+    <div class="leftcol">
+      <iframe id="fxa" scrolling="no" data-src="{{ settings.FXA_IFRAME_SRC }}?utm_campaign=fxa-embedded-form&amp;utm_medium=referral&amp;utm_source=firstrun&amp;utm_content=fx-{{ version }}&amp;entrypoint=firstrun&amp;service=sync&amp;context=iframe&amp;style=chromeless&amp;haltAfterSignIn=true"></iframe>
+    </div>
+
+    {#  copy right #}
+    <div class="rightcol">
```

Comment:
Just a thought: it would be nice if the copy came before the iframe in the source order, though I know that doesn't make floats easy. Since the iframe is a fixed width anyway, we could pad a container and position the iframe into the padded space, something like:

`.wrapper { position: relative; padding-left: 440px; }`
`.frame { width: 400px; position: absolute; left: 0; top: 0; }`

A bonus is that we could then let the content stack in small viewports (I know we're targeting 800px for now but this would let us accommodate even smaller).